torch/_subclasses/fake_tensor.py

```diff
@@ -905,6 +908,26 @@ def __init__(self, *args, **kwargs):
     def from_tensor(t, fake_mode):
         return fake_mode.from_tensor(t)
 
+    def clone_preserve_strides_storage(self):
+        with in_kernel_invocation_manager(self.fake_mode), torch.no_grad():
+            new_tensor = torch._prims.utils.clone_preserve_strides(self)
+            if self.requires_grad:
+                new_tensor.requires_grad = self.requires_grad
+                if not self.is_leaf:
+                    with torch.enable_grad():
+                        new_tensor = torch._prims.utils.clone_preserve_strides(self)
+                if self.grad is not None:
+                    self.grap = self.grad.clone_preserve_strides_storage()
```

Comment:
I'm wondering whether this piece of code is well exercised by the tests...
typo: `self.grap` -> `self.grad`